cves/CVE-2015-5346.yml

```diff
@@ -136,8 +137,8 @@ subsystem:
     directory names. Look at comments in the code. Look at the bug reports how
     the bug report was tagged.
     Examples: "clipboard", "gpu", "ssl", "speech", "renderer"
-  answer:
-  name:
+  answer: it lists the component in the bug report
+  name: Catalina
```

Comment:
This could probably be named Catalina Connector - Request or Catalina Connector - Coyote Adapter